module/Entra/Microsoft.Entra/Users/Get-EntraUserMembership.ps1

```diff
@@ -71,6 +71,9 @@ function Get-EntraUserMembership {
                 $params["All"] = $PSBoundParameters["All"]
             }
         }
+        elseif (-not $PSBoundParameters.ContainsKey("Top")) {
```

Comment:
Is it possible to add any additional logic to detect if the query is returning less objects than the default API limit, and only show the warning if there might be more?